src/data/text/item_descriptions.h

```diff
@@ -2793,3 +2793,9 @@ static const u8 sShinyCharmDesc[] = _(
 	"A charm that will\n"
     "raise the chance\n"
     "of Shiny Pokémon.");
+    
+static const u8 sOvalCharmDesc[] = _(
+    "Increases the\n"
+    "chance of finding\n"
+    "eggs at the daycare.");
```

Comment:
![pokeemerald-0](https://user-images.githubusercontent.com/2904965/96515170-39fa2f80-123b-11eb-96e3-46eaf3bc899c.png)
The text goes over the description area.

Reply:
suggestion: "Raises the chance" on the first line maybe?

Reply:
Increases the odds

Reply:
![image](https://user-images.githubusercontent.com/2904965/96515607-01a72100-123c-11eb-8fec-89d84aae863a.png)
That should work